packages/gcm/lib/gcm.dart

```diff
@@ -0,0 +1,10 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/// Service exposed to Flutter apps that implements a subset of the GCM API.
+///
+/// This library will probably be moved into a separate package eventually.
```

Comment:
(it doesn't have to be now, but it should be really soon. Having Google-specific services in the Flutter repo gives a very mixed message about what Flutter is. We want to make sure other companies and developers understand that they are peers, not subordinate in any way.)